test/unittests/librpc/test_StatisticProtocolServer.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * @CopyRight:
+ * FISCO-BCOS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FISCO-BCOS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FISCO-BCOS.  If not, see <http://www.gnu.org/licenses/>
+ * (c) 2016-2020 fisco-dev contributors.
+ */
+
+/**
+ * @brief : test_StatisticProtocolServer.cpp
+ * @author: yujiechen
+ * @date: 2020-07-08
+ */
+
+#include <libflowlimit/RPCQPSLimiter.h>
+#include <librpc/ModularServer.h>
+#include <librpc/Rpc.h>
+#include <librpc/StatisticProtocolServer.h>
+#include <test/tools/libutils/Common.h>
+#include <test/tools/libutils/TestOutputHelper.h>
+#include <boost/test/unit_test.hpp>
+#include <utility>
+
+using namespace dev;
+using namespace dev::rpc;
+using namespace dev::flowlimit;
+
+namespace dev
+{
+namespace test
+{
+class FakeStatisticProtocolServer : public StatisticProtocolServer
+{
+public:
+    FakeStatisticProtocolServer(size_t _fake)
```

Comment:
Codacy found an issue: [Class 'FakeStatisticProtocolServer' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5819294)